index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>N & K cakes</title>
+    <link rel="stylesheet" href="style.css">
+    <script  src="js/index.js"></script>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <div class="page-wrapper">
+    <!--HEADER-->
+<div id="home">
+        <div class="header" id="header">
+
+        <!--NAV-->
+        <div id="header-img">
+          <img src="images\logo.png" alt="">
+        </div>
+
+
+            <nav id="nav-bar">
+              <a href="#home" class="nav-link">Home</a>
+              <a href="#about" class="nav-link">About Us</a>
+              <a href="#contact" class="nav-link">Contact</a>
+            </nav>
+          </div>
+
+    <!--MAIN-->
+
+    </div>
+
+    <div class="about" id="about">
+      <h3>About our Cakes</h3>
+      <p>
+        We use the best flour, free-range eggs, and high quality butter for our Cakes. Additionally ingredients and flavors include dried, candied, or fresh fruit, nuts, cocoa, and extracts such as vanilla, with numerous substitutions for the primary ingredients.
+
+Are you a Vegan? We propose a glutten-free and sugar free Cakes and candies to make you happy!
+
+Our perfect dessert sauces are made off fresh fruits and nuts. Also you can pick a traditional icing like buttercream. Decorated with marzipan, piped borders, or candied our cakes will make your party unforgettable!
+      </p>
+      <div id="flexbox">
+        <img src="images\cake2.jpg" alt="">
+          <img src="images\cake3.jpg" alt="">
+      </div>
```

Comment:
Would be nice to look in the future and wrap your products items with a grid container. This will facilitate a fast and easy product section update and expansion while keeping this flexible and responsive. 😉